conan/tools/cmake/toolchain/blocks.py

```diff
@@ -328,6 +328,42 @@ class AppleSystemBlock(Block):
         # Setting CMAKE_OSX_DEPLOYMENT_TARGET if "os.version" is defined by the used conan profile
         set(CMAKE_OSX_DEPLOYMENT_TARGET "{{ CMAKE_OSX_DEPLOYMENT_TARGET }}" CACHE STRING "")
         {% endif %}
+        set(BITCODE "")
```

Comment:
Do we need to declare these variables with empty?

Reply:
I thought it was not needed too but one test failed because of that. I'll recheck it just in case

Reply:
That was the reason: https://github.com/conan-io/conan/pull/10292/files#diff-a42d08ebe9e9d3240230bce3ce865eb2b03967fc334ab18ba057bda007990e8aR183

Reply:
I don't get it.